web/gds-admin-ui/src/routes/PrivateRoute.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Route, Redirect } from 'react-router-dom';
+
+import { APICore } from '../helpers/api/apiCore';
+
+
+const PrivateRoute = ({ component: Component, ...rest }) => {
+    const api = new APICore();
```

Comment:
In the future we should think about having simple state like the `user` and `isAuthenticated` stored in redux state rather than in session storage accessible only through the `APICore` functions. I don't believe that `user` and `isAuthenticated` are secure information, and they can be loaded from the session store using a reducer or an action. That might simplify this very common requirement to get access to the user state. What do you think? 

If you agree, please create a story in the backlog and we can work on it in the future!